internal/common/splunk/common.go

```diff
@@ -21,6 +21,7 @@ const (
 	SFxAccessTokenLabel   = "com.splunk.signalfx.access_token"
 	SFxEventCategoryKey   = "com.splunk.signalfx.event_category"
 	SFxEventPropertiesKey = "com.splunk.signalfx.event_properties"
+	SourcetypeLabel       = "source.type"
```

Comment:
Can we use "com.splunk.sourcetype" instead?

Reply:
In this case we cannot - "source.type" is a field that is part of the Splunk log payload. It is defined here:
https://docs.splunk.com/Documentation/Splunk/8.0.6/Data/FormateventsforHTTPEventCollector

But you just made me realize this should be "sourcetype". The period snuck in. I'll fix that.

Reply:
I believe the definition in the spec is incorrect. I wrote it before Otel introduced guidelines for conventions which say companies need to use reverse domain name as a prefix: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/common/common.md#recommendations-for-application-developers

>The name is specific to your company and may be possibly used outside the company as well. To avoid clashes with names introduced by other companies (in a distributed system that uses applications from multiple vendors) it is recommended to prefix the new name by your company's reverse domain name, e.g. com.acme.shopname.

Reply:
Makes sense, making the change.